config/routes.rb

```diff
@@ -5,6 +5,7 @@
 
   resource :home, only: :show
   resource :about, only: :show, controller: :about
+  resource :expenses, only: :show, path: "income-report"
```

Comment:
If the route is `/income-report` then the controller should be named the same or similar.

Reply:
I'm happy with both being "financials".